pom.xml

```diff
@@ -23,25 +23,26 @@
 
   <properties>
     <java.version>11</java.version>
-    <folio-spring-base.version>1.0.0</folio-spring-base.version>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+    <folio-spring-base.version>1.0.5</folio-spring-base.version>
     <openapi-generator.version>4.3.1</openapi-generator.version>
     <openapi.input.file>${project.basedir}/src/main/resources/swagger.api/inn-reach.yaml</openapi.input.file>
-    <mapstruct.version>1.3.1.Final</mapstruct.version>
-    <json.version>20200518</json.version>
+    <mapstruct.version>1.4.2.Final</mapstruct.version>
     <sonar.exclusions>
       src/main/java/org/folio/innreach/controller/TenantController.java,
       src/main/java/org/folio/innreach/ModInnReachApplication.java,
       src/main/java/org/folio/innreach/domain/entity/**,
-      src/test/**
+      src/main/java/org/folio/innreach/repository/**
```

Comment:
Could you please explain why we need to use this pattern **src/main/java/org/folio/innreach/repository/****
 instead of the previous one?

Reply:
test are ignored by default, so you don't need to mention them at all
i've added repository because usually a repository is an interface, thus there is nothing to cover really